index.html

```diff
@@ -1,45 +1,49 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <style>
-      textarea {
-        font-family: monospace;
-        display: block;
-      }
-      textarea:disabled {
-        color: black;
-      }
-    </style>
-    <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.48.2/codemirror.min.css">
+    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.48.2/codemirror.min.css">
   </head>
   <body>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.48.2/codemirror.min.js"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.48.2/mode/javascript/javascript.min.js"></script>
+    <dialog id="loading">
+      Loading...
+    </dialog>
+    <script>
+      loading.showModal();
+      window.addEventListener('error', () => {
+        loading.close();
+      });
+    </script>
+    <!-- github ribbon -->
+    <a id="github" href="https://github.com/engine262/engine262"><img width="149" height="149" src="https://github.blog/wp-content/uploads/2008/12/forkme_right_darkblue_121621.png?resize=149%2C149" class="attachment-full size-full" alt="Fork me on GitHub" data-recalc-dims="1"></a>
```

Comment:
Can this be an embedded SVG or something? i think http://tholman.com/github-corners/ is where people get ribbons these days.

Reply:
I'm neutral but kind of like that this one at least says in text what it is about to do.

Reply:
copying the png into the repo would also work i guess. i just don't want to be loading things off of random sites.

Reply:
vendored into `github_ribbon.png`